sdk/core/core-client-rest/test/sendRequest.spec.ts

```diff
@@ -83,6 +83,29 @@ describe("sendRequest", () => {
         contentType: "multipart/form-data",
       });
     });
+
+    it("should handle request body as FormData with array of binary", async () => {
```

Comment:
Could you add a case to customize the filename and content type for arrays of files?

How about two arrays?

Reply:
I will add more test cases (multiple arrays is a good idea), but FWIW this code doesn't handle file name or content type at all -- that stuff just gets passed through to `core-rest-pipeline` transparently as they are properties on the `Blob`/`File` object.

Reply:
Added a more complex test case with multiple arrays of files, text, etc.